framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInference.java

```diff
@@ -120,17 +122,20 @@ void updateInferredFieldType(
 
     /**
      * Updates the return type of the method {@code methodTree} based on {@code returnedExpression}.
+     * May also strengthen the return types of any abstract methods that this method overrides.
      *
      * @param returnedExpression the node that contains the expression returned
      * @param classSymbol the symbol of the class that contains the method
      * @param methodTree the tree of the method whose return type may be updated
+     * @param overriddenMethods the methods that the given method return overrides
```

Comment:
The parameter's type is a map type, but this documentation doesn't explain what it maps from and to.